fastestimator/op/tensorop/meta/repeat.py

```diff
@@ -30,6 +30,25 @@
 @traceable()
 class Repeat(TensorOp):
     """Repeat a TensorOp several times in a row.
+    Repeat takes an Op and runs it multiple times in a row. It can be set to repeat for a fixed (static) number of
+    times, or to repeat until a given input function evaluates to False (dynamic).
+
+    Static example:
+
+        ops=[
+            LambdaOp(fn=lambda: 0, outputs="z"),
+            Repeat(AddOne(inputs="z", outputs="z"), repeat=5)
+            ]
+
+    Dynamic example:
+
+        ops=[
+            LambdaOp(fn=lambda: 0, outputs="z"),
+            Repeat(AddOne(inputs="z", outputs="z"), repeat=lambda z: z < 6.5)
+            ]
+
+        Note : Here the argument ('z') of the lambda function used as repeat callable function is the key used by the
+               ops passed to the Repeat Op.
 
     Args:
         op: A TensorOp to be run one or more times in a row.
```

Comment:
In line 58 we have that "If a function is provided, any TF ops which are wrapped by Repeat will not have access to the gradient tape, nor to previously deferred model update functions."

Is that still true? Does it now also apply to repeating N number of times?